doc/content/the-things-stack/monitor/gateway-connectivity-outages/_index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Gateway connectivity outages"
```

Comment:
This `gateway-connectivity-outages` folder should be under the `alerting` folder for the correct hierarchy.